health/micro-ui/web/packages/modules/campaign-manager/src/pages/employee/NewAppConfiguration/components/DropdownTemplate.js

```
@@ -1,32 +1,54 @@
+        return { moduleName: null, masterName: null, isValidSchema: false };
+      }
+
+      return {
+        moduleName: schemaParts[0]?.trim(),
+        masterName: schemaParts[1]?.trim(),
+        isValidSchema: true,
+      };
+    }, [selectedField?.isMdms, selectedField?.schemaCode]);
+
+    // Fetch MDMS data if field has valid MDMS configuration
+    const { isLoading, data: mdmsData } = Digit?.Hooks.useCustomMDMS(
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)